include/circt/Dialect/SV/SVVisitors.h

```diff
@@ -30,15 +30,16 @@ class Visitor {
             // Expressions
             ReadInOutOp, ArrayIndexInOutOp, VerbatimExprOp, VerbatimExprSEOp,
             IndexedPartSelectInOutOp, IndexedPartSelectOp, StructFieldInOutOp,
-            ConstantXOp, ConstantZOp, MacroRefExprOp, MacroRefExprSEOp,
+            ConstantXOp, ConstantZOp, ConstantStrOp, MacroRefExprOp,
+            MacroRefExprSEOp,
```

Comment:
Random: it'd be great if this could be created for us from TableGen, so we didn't have to maintain it!

Reply:
I was thinking of adding an expr trait.

Reply:
There is generally a lot of redundancy in those checks left over from spilling.